### ISSUE TITLE
Bug: mojeek engine

### ISSUE THREAD

**Issue by nicfab** [searxng/searxng#1828]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**
Repository: https://github.com/searxng/searxng
Branch: master
Version: 2022.09.29-a7d69323

**How did you install SearXNG?**
I installed SearXNG via scripts
**What happened?**
After having launched a search I receive that error.

**How To Reproduce**
See the screenshot attached.

**Expected behavior**
No errors

**Screenshots & Logs**
I attached two screenshots.

**Additional context**
I have the following settings in `/etc/searxng/settings.yml` 

```
# SearXNG settings

use_default_settings: true

general:
  debug: false
  instance_name: "SearXNG NicFab"
  contact_url: "mailto:info@nicfab.it"
  privacypolicy_url: https://docs.searxng.org/own-instance.html#how-does-searxng-protect-privacy

search:
  safe_search: 2
  autocomplete: 'duckduckgo'

server:
  secret_key: "redacted"
  limiter: true
  image_proxy: true

redis:
  url: unix:///usr/local/searxng-redis/run/redis.sock?db=0

ui:
  static_use_hash: true

# preferences:
#   lock:
#     - autocomplete
#     - method

enabled_plugins:
  - 'Hash plugin'
  - 'Search on category select'
  - 'Self Informations'
  - 'Tracker URL remover'
  - 'Ahmia blacklist'
  # - 'Hostname replace'  # see hostname_replace configuration below
  # - 'Infinite scroll'
  # - 'Open Access DOI rewrite'
  # - 'Vim-like hotkeys'

# plugins:
#   - only_show_green_results

# hostname_replace:
#
#   # twitter --> nitter
#   '(www\.)?twitter\.com$': 'nitter.net'

engines:
  - name: google
    use_mobile_ui: true

  - name: mojeek
    disabled: false

  - name: bing
    disabled: false

  - name: duckduckgo
    disabled: false

  - name: qwant
    disabled: false

  - name: yahoo
    disabled: false

  - name: petalsearch
    disabled: false


#   - name: fdroid
#     disabled: false
#
#   - name: apk mirror
#     disabled: false
#
#   - name: mediathekviewweb
#     categories: TV
#     disabled: false
#
#   - name: invidious
#     disabled: false
#     base_url:
#       - https://invidious.snopyta.org
#       - https://invidious.tiekoetter.com
#       - https://invidio.xamh.de
#       - https://inv.riverside.rocks
```

**Technical report**
**Exception** | searx.exceptions.SearxEngineAccessDeniedException | Percentage | 25
-- | -- | -- | --
**Parameter** | HTTP error 403
**Filename** | searx/engines/xpath.py:199
**Function** | response
**Code** | raise_for_httperror(resp)

![CleanShot 2022-09-27 at 18 30 22](https://user-images.githubusercontent.com/24752671/193233312-511c54ee-1d56-400a-8988-5dbfe93434d1.png)
![CleanShot 2022-09-30 at 10 49 42](https://user-images.githubusercontent.com/24752671/193233320-c9d3b2cc-a37c-4720-8756-a1e24551489e.png)

**Comment by tiekoetter** [comment]
Your IP got blocked by mojeek. This is expected to happen.

You can try setting up some proxy servers to distribute the load.

**Comment by nicfab** [comment]
> Your IP got blocked by mojeek. This is expected to happen.
> 
> You can try setting up some proxy servers to distribute the load.

Thank you. 
I am in contact with Mojeek to investigate and solve this issue.

**Comment by tiekoetter** [comment]
I don't think they allow their search engine to be scraped by an external engine. So them blocking your instance (or any other instance) is well in their right to do.

If you can manage to get something from them regarding rate limiting it would be great if you would share it here.

**Comment by nicfab** [comment]
> I don't think they allow their search engine to be scraped by an external engine. So them blocking your instance (or any other instance) is well in their right to do.
> 
> If you can manage to get something from them regarding rate limiting it would be great if you would share it here.

If I have any information or solution from Mojeek, I will return here to share it with you. Indeed.

**Comment by Shituation** [comment]
Take my friendly advice. Move on to other search engines. These people censores you.

**Comment by JReming85** [comment]
> - name: mojeek
>     disabled: false

Do you know the proper format to enable a proxy per engine?

I tried the documented 

```engines:
  - name: google
    engine: google
    proxies:
      http:
        - http://<rotating ipv6 proxy>

  - name: mojeek
    disabled: false
    proxies:
      http:
        - http://<rotating ipv4 proxy>```
but I dont think its working

**Comment by return42** [comment]
I assume this issue is closed by:

- https://github.com/searxng/searxng/pull/2266

If not please lets open a new issue.

**Comment by nicfab** [comment]
> I assume this issue is closed by:
> 
> * [[mod] Shuffle httpx's default ciphers of a SSL context randomly. #2266](https://github.com/searxng/searxng/pull/2266)
> 
> If not please lets open a new issue.

I continue having the error.
